app/src/main/resources/logging.properties

```diff
@@ -0,0 +1,7 @@
+handlers= java.util.logging.FileHandler
+
+# File Logging
+java.util.logging.FileHandler.pattern = %h/.scenebuilder/logs/scenebuilder-${version}.log
```

Comment:
this fails on Mac:
```
Can't load log handler "java.util.logging.FileHandler"
java.nio.file.NoSuchFileException: /Users/JosePereda/.scenebuilder/logs/scenebuilder-16.0.0-SNAPSHOT.log.lck
```

If you check AppPlatform, the app folder is not `~/.scenebuilder` for all platforms.

Reply:
I now see that the [Application Data directory](https://github.com/gluonhq/scenebuilder/blob/master/app/src/main/java/com/oracle/javafx/scenebuilder/app/AppPlatform.java#L64) is different for different platforms. I wonder why was it done instead of using `${user.home}/.scenebuilder`?

Reply:
Can't reply to that :) 
If needed for logs only you might want to simply use `${user.home}/.scenebuilder` for all platforms?